docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,140 @@
+= Getting Started
+
+Everyone has to start somewhere and if you’re new to VIVIDUS, this is where to begin.
+
+== Before you start
+In order to use VIVIDUS effectively, you need to know what it is and understand
+some of its fundamental concepts. So before you start using VIVIDUS in earnest,
+we highly recommend you read xref:ROOT:tests-configuration.adoc["How to cook VIVIDUS?"].
+
+== Prerequisites
+
+=== Install and configure Git
+. Download and install http://git-scm.com/downloads[Git client]
```

Comment:
It's not really required, isn't it? :)

Reply:
that's true, I thought about this point, but decided to keep it as is in a sake of simplicity